packages/dtslint/src/updateConfig.ts

```diff
@@ -15,7 +15,6 @@ import { Configuration as Config, ILinterOptions, IRuleFailureJson, Linter, Lint
 import * as ts from "typescript";
```

Comment:
Is this script needed anymore? Does it work without tslint?

Reply:
I guess we aren't removing tslint in this PR? That's fine too. Cleanup can be later.

Reply:
Yeah my strategy guess was to get this one in minimally and then do a full TSLint removal. That'd be my preference. But I'm flexible :)